jakartaee10/JakartaEE10ReleasePlan.md

```diff
@@ -4,3 +4,167 @@
+
+This is a run-at goal for Jakarta EE 10.
+We know that not all Specifications can establish their standalone TCKs in this timeframe.
+The Platform TCK can initiate this refactoring by breaking apart the TCK source into separate directories within the Platform TCK repository.
+This will help with the eventual move of these TCK directories into their respective component repositories.
+
+## Component Specification Project Releases
+As mentioned, many of the Component Specification projects have already declared their intent for either a Major or Minor version update for Jakarta EE 10.
+These projects have already submitted or completed their Plan Reviews for inclusion in Jakarta EE 10.
+These updates are outlined in the sections below.
+
+## Deliverables in Jakarta EE 10
+The following details the specifications and APIs currently planned for the Jakarta EE 10 Platform and Web Profile.
+
+### Jakarta EE 10 Specifications
+List of specifications in Jakarta EE 10 Platform and Web Profile (updated specifications marked with asterisks).
+
+* Jakarta EE Platform 10*
+* Jakarta EE Web Profile 10*
+* Jakarta Activation 2.1*
+* Jakarta Annotations 2.0 (Web Profile)
+* Jakarta Authentication 3.0* (Web Profile)
+* Jakarta Authorization 3.0*
+* Jakarta Batch 2.1*
+* Jakarta Bean Validation 3.0 (Web Profile)
+* Jakarta Concurrency 3.0* (Web Profile)
+* Jakarta Connectors 2.1*
+* Jakarta Contexts and Dependency Injection 4.0*
+* Jakarta Debugging Support for Other Languages 2.0 (Web Profile)
+* Jakarta Dependency Injection 2.0 (Web Profile)
+* Jakarta Enterprise Beans 4.0
+* Jakarta Enterprise Beans 4.0 Lite (Web Profile)
+* Jakarta Enterprise Web Services 2.0 (Optional)
+* Jakarta Expression Language 5.0* (Web Profile)
+* Jakarta Interceptors 2.0 (Web Profile)
+* Jakarta JSON Binding 2.1* (Web Profile)
+* Jakarta JSON Processing 2.1* (Web Profile)
+* Jakarta Mail 2.1*
+* Jakarta Managed Beans 2.0 (Web Profile)
+* Jakarta Messaging 3.1*
+* Jakarta Persistence 3.1* (Web Profile)
+* Jakarta RESTful Web Services 3.1* (Web Profile)
+* Jakarta Security 3.0* (Web Profile)
+* Jakarta Server Faces 4.0* (Web Profile)
+* Jakarta Server Pages 3.1* (Web Profile)
+* Jakarta Servlet 6.0* (Web Profile)
+* Jakarta SOAP with Attachments 3.0* (Optional)
+* Jakarta Standard Tag Library 3.0* (Web Profile)
+* Jakarta Transactions 2.0 (Web Profile)
+* Jakarta WebSocket 2.1* (Web Profile)
+* Jakarta XML Binding 4.0* (Optional)
+* Jakarta XML Web Services 4.0* (Optional)
+ * Jakarta Web Services Metadata 3.0 (Merged)
+
+### Proposed Updates to Web Profile
+* Move Concurrency 3.0* from Platform to Web Profile 10* ([issue](https://github.com/eclipse-ee4j/jakartaee-platform/issues/363#issuecomment-846424050))
+
+### Jakarta EE 10 APIs
+The Platform and Web Profile API uber jar files will be re-generated to correspond with the Jakarta EE 10 release designation.
```

Comment:
I think we need to mention that these will not be modularized since it conflicts with the individual specification apis jars. It is implied from above, but should be explicit. Vendors are also free to provider their own versions with module-info.class as appropriate.

Reply:
Sure, I can be more explicit in this section.  Thanks.